shared/services/rocketpool/assets/install/alerting/alertmanager.tmpl

```diff
@@ -40,13 +40,25 @@ receivers:
     discord_configs:
       - webhook_url: "{{ .DiscordWebhookURL.Value }}"
     {{- end }}
-  
+
+    {{- if .PushoverToken.Value and .PushoverUserKey.Value }}
+    pushover_configs:
+      - token: "{{ .PushoverToken.Value }}"
+        user_key: "{{ .PushoverUserKey.Value }}"
+    {{- end }}
+
   - name: "node_operator_no_resolved"
     {{- if .DiscordWebhookURL.Value }}
     discord_configs:
       - webhook_url: "{{ .DiscordWebhookURL.Value }}"
         send_resolved: false
     {{- end }}
+    
+    {{- if .PushoverToken.Value .PushoverUserKey.Value }}
```

Comment:
Was it intentional to repeat this code here?

Reply:
the idea was to mimic the same block as with discord, but I see that I copied pasted and forgot to add `send_resolved: false`. Now fixed, thanks for catching it.